tests/ignite/distributed/check_idist_parallel.py

```diff
@@ -38,14 +38,29 @@ def training(local_rank, config, **kwargs):
         python tests/ignite/distributed/check_idist_parallel.py
         ```
 
-        - Launch 4 procs using gloo backend with `torch.distributed.launch`
+        - Launch 4 procs using gloo backend with `torchrun` or `torch.distributed.launch`:
```

Comment:
Let's just replace the old method by the new one:
```suggestion
        - Launch 4 procs using gloo backend with `torchrun`:
```